service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerSubmittedTest.java

```diff
@@ -364,34 +387,39 @@ void shouldUpdateRepresentativesAccess() {
             .map(changeRequest -> Map.of("changeOrganisationRequestField", changeRequest))
             .collect(toList());
 
+        final List<String> expectedCaseEvents = List.of("internal-update-case-summary",
+            "internal-change-UPDATE_CASE", "updateRepresentation");
+
+        verify(concurrencyHelper, timeout(ASYNC_METHOD_CALL_TIMEOUT).times(5))
+            .startEvent(eq(caseData.getId()), any());
+
         // setup change field, submit to aac (org1), setup change field, submit to aac (org2), update case-summary
-        verify(concurrencyHelper, times(5)).submitEvent(any(),
-            eq(caseData.getId()), caseCaptor.capture()
-        );
+        checkThat(() -> {
+            verify(concurrencyHelper, times(5)).submitEvent(startEventCaptor.capture(),
+                eq(caseData.getId()), caseCaptor.capture()
+            );
+        }, Duration.ofSeconds(10));
```

Comment:
Was there any issues with the checkThat in combination with the times(5) thing, I think we chained them before right, `times(5).timeout(10000)` ?

Reply:
yeah for this instance the standard times(5) one failed so swapped it back to the checkThat()